contentcuration/contentcuration/management/commands/loadconstants.py

```diff
@@ -94,7 +94,6 @@ def get_dict(self, constant):
 class FormatGenerator(ConstantGenerator):
     module = file_formats
     filename = "formatlookup.json"
-    default_list = file_formats.FORMATLIST
```

Comment:
@jayoshih  I removed these `default_list` attributes since they are not used anywhere in the code.